app/src/main/java/com/example/cm/ui/meetup/MeetupDetailed/MeetupDetailedFriendsListFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.example.cm.ui.meetup.MeetupDetailed;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+
+import com.example.cm.databinding.FragmentMeetupDetailedFriendsListBinding;
+import com.example.cm.ui.adapters.MeetupDetailedFriendListAdapter;
+
+import java.util.List;
+
+public class MeetupDetailedFriendsListFragment extends Fragment {
+
+    private final List<String> friends;
+    private FragmentMeetupDetailedFriendsListBinding binding;
+
+    public MeetupDetailedFriendsListFragment(List<String> friends) {
+        this.friends = friends;
+    }
+
+    private void initUI() {
+        binding.meetupDetailedFriendsList.setLayoutManager(new LinearLayoutManager(getContext()));
+        binding.meetupDetailedFriendsList.setHasFixedSize(true);
+    }
+
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        binding = FragmentMeetupDetailedFriendsListBinding.inflate(inflater, container, false);
+
+        initUI();
+        initViewModel();
+
+        return binding.getRoot();
+    }
+
+    private void initViewModel() {
+        MeetupDetailedFriendsListViewModel viewModel = new ViewModelProvider(this, new MeetupDetailedFriendsListFactory(friends)).get(MeetupDetailedFriendsListViewModel.class);
+        viewModel.getUsers().observe(getViewLifecycleOwner(), users -> {
```

Comment:
Mayb rename the getUsers to getFriends(). Was a bit confused whether all users would get retrieved or just the actual friends.

Reply:
I am not sure how we want to handle this. There could be user that are not on your friend list, e.g. a friend created the meeting and invited his friends which are not friends with you..